includes/posting.inc.php

```diff
@@ -1389,9 +1389,6 @@
      $smarty->assign('spam',intval($field['spam']));
      $smarty->assign('spam_check_status',intval($field['spam_check_status']));
      $smarty->assign('back',$back);
-     #$smarty->assign('page',$page);
```

Comment:
Are these assignments superfluous? I thik, I saw them at many places in the code.